python/swe/dockerfiles/create_index.py

```diff
@@ -11,6 +11,8 @@
 from composio.utils.logging import WithLogger
 
 
+print("This is a Test PR")
```

Comment:
If this change is intended for testing the PR agent, it might be more appropriate to add a new test file in a dedicated test directory, rather than modifying a core module. This would allow for more comprehensive testing without affecting the main codebase.

The added print statement doesn't seem to relate to the main functionality of this module, which is about creating Docker indexes. If this is a temporary change for testing purposes, please ensure it's removed before merging to the main branch.

If this change is intended to remain (which is not recommended), please add a comment explaining its purpose and when it can be removed. For example:
```python
# TODO: Remove this print statement after PR agent testing is complete
print('This is a Test PR')
```